### DISCUSSION TITLE
General Usage

### DISCUSSION THREAD

**Discussion by thomas-cristo-lekker** [patrickkusebauch/deptracpy#8] [General]
Hey,
I already use deptrac  for PHP sometimes and now I try deptracpy. But I got only uncovered source file warnings, when I run deptracpy. 
Is anything running before? mypy?

**Comment by thomas-cristo-lekker** [comment]
Ah, I have to define my own ruleset. Now I got it:
```
┏━━━━━━━━━━━━┳━━━━━━━┓
┃            ┃ Count ┃
┡━━━━━━━━━━━━╇━━━━━━━┩
│ Allowed    │ 1940  │
│ Warning    │ 616   │
│ Violations │ 17    │
│ Errors     │ 0     │
└────────────┴───────┘
```

**Comment by patrickkusebauch** [comment]
Yes, the usage is very similar to `deptrac` intentionally. It is just not as feature-rich as `deptrac` is at the moment.

**Comment by patrickkusebauch** [comment]
@thomas-cristo-lekker BTW if you have feature requests, feel free to spam the issues with them, I am happy to implement them.
